hack/validate-blocked-edges.py

```diff
@@ -8,15 +8,20 @@
 def validate_blocked_edges(directory):
     for path, data in util.walk_yaml(directory=directory, allowed_extensions=('.yaml',)):
         try:
-            validate_blocked_edge(data=data)
+            validate_blocked_edge(data=data, path=path)
         except Exception as error:
-            raise ValueError('invalid blocked edge {}: {}'.format(path, error))
+            raise ValueError('invalid blocked edge {}: {}'.format(path, error)) from None
```

Comment:
This is unrelated to the fix, but results in less clutter. It prevents Python from writing out a `During handling of the above exception, another exception occurred` part with a second traceback for the original exception.